doc/supportedPlatformVersions.md

```diff
@@ -2,6 +2,52 @@
 
 ## dotnet
 
+### bookworm
+
+- 6.0.100-preview.1.21103.13
+- 6.0.100-preview.2.21155.3
+- 6.0.100-preview.3.21202.5
+- 6.0.100-preview.4.21255.9
+- 6.0.100-preview.5.21302.13
+- 6.0.100-preview.6.21355.2
+- 6.0.100-preview.7.21379.14
+- 6.0.100-rc.1.21458.32
+- 6.0.100-rc.2.21505.57
+- 6.0.100-rtm.21524.1
+- 6.0.100
+- 6.0.201
+- 6.0.300
+- 6.0.301
+- 6.0.400
+- 6.0.401
+- 6.0.403
+- 6.0.404
+- 6.0.405
+- 6.0.406
+- 6.0.407
+- 6.0.408
+- 6.0.410
+- 6.0.411
+- 6.0.412
+- 7.0.100-preview.1.22110.4
+- 7.0.100-preview.5.22307.18
+- 7.0.100-preview.6.22352.1
+- 7.0.100-preview.7.22377.5
+- 7.0.100-rc.1.22431.12
+- 7.0.100-rc.2.22477.23
+- 7.0.100
+- 7.0.101
+- 7.0.102
+- 7.0.201
+- 7.0.202
+- 7.0.203
+- 7.0.302
+- 7.0.304
+- 7.0.305
+- 7.0.306
+- 8.0.100-preview.6.23330.14
+- 8.0.100-preview.7.23376.3
```

Comment:
I am wondering how you decided which dotnet versions to support.

Reply:
https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/releases-index.json